ui/job-view/details/BugFiler.jsx

```diff
@@ -264,25 +264,27 @@ export class BugFilerClass extends React.Component {
       if (lowerJobGroupName.includes('web platform')) {
         failurePath = failurePath.startsWith('mozilla/tests')
           ? `testing/web-platform/${failurePath}`
-          : `testing/web-platform/tests/${failurePath}`;
+          : `testing/web-platform/tests${failurePath}`;
       }
 
       // Search mercurial's moz.build metadata to find products/components
       fetch(
         `${hgBaseUrl}mozilla-central/json-mozbuildinfo?p=${failurePath}`,
       ).then(resp =>
         resp.json().then(firstRequest => {
+          if (firstRequest.error) {
+            notify("Unable to infer product/component via metadata. Please manually search for a product.");
+          }
           if (
-            firstRequest.data.aggregate &&
+            firstRequest.data && firstRequest.data.aggregate &&
```

Comment:
firstRequest.data was undefined in the case where moz-build returns an error instead of data.